crates/types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-operator-types"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
Super Nit: _Technically_ this should be 0.2.0 since you're adding new features

Reply:
you're absolutely right, this should probably be a 0.2.0, I think we've broken the rule about patch versions a number of times already, so I think we'll probably want to have that conversation when we do the work to [move to v1alpha2](https://github.com/wasmCloud/wasmcloud-operator/issues/41).